travis/build-all-manylinux-wheels.sh

```diff
@@ -3,8 +3,14 @@
 set -x
 set -e
 
+ARCH=$1
+
 # Wait for docker pull to complete downloading container
-manylinux_image="ghcr.io/pyca/cryptography-manylinux2014:x86_64"
+f [ $ARCH = 'aarch64' ]; then
```

Comment:
There's a typo here. It should be `if` not just `f`

Reply:
Done.